docs/reference/api/pytorch.txt

```diff
@@ -170,6 +170,18 @@ your code:
    to the ``clip_grads`` argument of ``step_optimizer``
    while removing the ``PyTorchCallback`` counterpart in the :meth:`build_callbacks` method.
 
+Debugging FAQ
+-------------
+
+* **Q: If my job got paused (e.g. if a higher weight job gets scheduled over the top of a running job),
+  then the resulting validation accuracies are significantly lower than for jobs that weren't paused,
+  even though the training accuracies are the same.**
+
+  * A: Please check if you have wrapped your model, optimizer, and LR scheduler with
```

Comment:
How about something a bit more organized, like:

```
  * A: Please verify the following:
          * your model is wrapped with ``wrap_model``
          * your optimizer is wrapped with ``wrap_optimizer`` and is based on output of ``wrap_model``, not the original unwrapped model
          * your LR scheduler is wrapped with ``wrap_lr_scheduler`` and is based on the output of ``wrap_optimizer``, not the original unwrapped optimizer
```

(you'll have to edit it for reST syntax and stuff)